requirements.txt

```diff
@@ -6,6 +6,7 @@ jishaku
 git+https://github.com/Rapptz/asqlite
 discord.py>=2.0.0
 git+https://github.com/Rapptz/discord-ext-menus
+git+https://github.com/oliver-ni/discord-ext-menus-views
```

Comment:
Is there a reason this dependency is needed? I've been trying to push down on the amount of dependencies and their file size recently, so I'd like some extra explanation for this
From what I can see this is mostly just a convenience thing and this could be done without said dependency